fibers/web/server.scm

```diff
@@ -176,6 +173,35 @@ on the procedure being called at any particular time."
               ((0) (memq 'keep-alive (response-connection response)))))
            (else #f)))))
 
+(define* (make-chunked-output-port* port #:key (keep-alive? #f)
+                                    (buffering 1200))
+  "This is a variant of make-chunked-output-port from (web http) which
+importantly sends a \r\n right at the end of the output.  Without
```

Comment:
That sounds like a Guile bug, for which there's a patch? I suppose guile-fibers could work around Guile bugs, but this sounds more like a distro's job to me (i.e., updating to new fixed version, or backporting the fix).

Reply:
It's been fixed in Guile now [1] but that fix hasn't been released yet.

1: https://git.savannah.gnu.org/cgit/guile.git/commit/?id=9a3353a993e8b73312a0005278f12c67cffa9854

Thinking about this more, maybe it is fine to not try to work around this here. I'll remove this procedure.